src/lib/blueprint.ts

```diff
@@ -51,6 +52,13 @@ export interface Resource {
   draftMessage: string
 }
 
+interface EventResource extends Resource {
+  resourceType: 'event'
+  eventType: string
+  routePath: string
```

Comment:
What do you think about adding this to `Resource`. Then, for each resource and event, we add

```
---
route_path: ...
---
```


This would remove the need to add the resources here: https://github.com/seamapi/docs/blob/main/src/data/paths.yaml#L9

Reply:
544602542a0a506b36afb24f4bb7416e9bbbe6c5